test/helpers/position_assertions.cc

```diff
@@ -82,26 +82,26 @@ string prettyPrintRangeComment(string_view sourceLine, const Range &range, strin
                        string(numLeadingSpaces + sourceLineNumber.length() + 1, ' '), string(numCarets, '^'), comment);
 }
 
-string_view getLine(const UnorderedMap<string, shared_ptr<core::File>> &sourceFileContents, string_view uriPrefix,
-                    const Location &loc) {
-    auto filename = uriToFilePath(uriPrefix, loc.uri);
+string_view getLine(const LSPConfiguration &config,
```

Comment:
These changes all involve no longer passing through `uriPrefix` and instead using `LSPConfiguration`.